mods/ra/rules/defaults.yaml

```diff
@@ -145,6 +145,24 @@
 		RequiresCondition: rank-elite
 		ZOffset: 256
 
+^InfantryExperienceHospitalOverrides:
+	WithDecoration@RANK-1:
+		BlinkInterval: 32
+		BlinkPatterns:
+			hospitalheal: True, False
```

Comment:
This is really bound to confuse modders. In my opinion it would be more modder-friendly to introduce a `Blink` enum with `On`/ `Off` and map that to True/False internally.

Reply:
What if I rename the fields to `VisibilityInterval`, `VisibilityPattern`, `VisibilityPatterns` instead?

Reply:
Wouldn't help, Blink sounds more reasonable here. I grasped the syntax now and tbh I don't think renaming this would help. It needs more detailed descriptions IMO.